src/blocks/popup.css

```diff
@@ -59,14 +59,15 @@
   line-height: 29px;
   letter-spacing: 0em;
   text-align: left;
-  margin: 0 0 50px 0;
+  margin: 0 0 30px 0;
 }
 
```

Comment:
Needs correction: the cursor of the **disabled** button should not be `pointer`, use `default`